functions/go/export-terraform/terraformgenerator/templates/log-export.tf

```diff
@@ -45,4 +45,14 @@ module "{{ .GetResourceName }}-destination" {
     is_locked             = {{ .GetBool "spec" "retentionPolicy" "isLocked" }}
   }{{end}}
 }
+{{end}}{{with $logsink.References.LoggingLogBucket }}
+module "{{ .GetResourceName }}-destination" {
+  source  = "terraform-google-modules/log-export/google//modules/logbucket"
+  version = "~> 7.3.0"
+
+  project_id                  = module.{{ .Parent.GetResourceName }}.project_id
+  log_bucket_name             = "{{ .GetResourceName }}"
```

Comment:
Shouldn't this be name?
https://github.com/terraform-google-modules/terraform-google-log-export/blob/eb777cae1f7a7ef7df4f7f9651797b0213b3412f/modules/logbucket/variables.tf#L22

Reply:
Yes, you're right.